tests/pytorch/mock_sparse/test_sddmm.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
```

Comment:
Let's run isort and black to polish our code.

https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit

Reply:
Done.